src/main/scala/org/phenoscape/kb/AnatomicalEntity.scala

```diff
@@ -128,14 +128,15 @@ object AnatomicalEntity {
     val valuesList = terms.map(t => sparql" $t ").fold(sparql"")(_ + _)
     sparql"""
             PREFIX hint: <http://www.bigdata.com/queryHints#>
+            PREFIX ps: <http://purl.org/phenoscape/vocab.owl#>
             SELECT DISTINCT ?x ?y
             FROM $KBClosureGraph
             FROM $KBMainGraph
             WITH {
               SELECT ?term ?presence
               WHERE {
                 VALUES ?term { $valuesList }
-                ?presence <http://purl.org/phenoscape/vocab.owl#implies_presence_of_some> ?term .
+                ?presence ps:implies_presence_of ?term .
```

Comment:
Embed the constant IRIs using the interpolator rather than managing prefixes yourself.